CONP_DATS_fields.md

```diff
@@ -1,47 +1,53 @@
+|Date|Primary Publications -> Dates||Relevant dates for the publication. If you provide a date, it must come with a description of the date.|Relevant dates for the publication. If you provide a date, it must come with a description of the date (i.e.: first submission, final approval, date of publication, ...).|
+|Identifier|Primary Publications -> Identifier|| Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the publication. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Identifier|Primary Publications -> Identifier -> Identifier|**RECOMMENDED**| A code uniquely identifying an entity locally to a system or globally.|A code uniquely identifying an entity locally to a system or globally.|
+|Identifier Source|Primary Publications -> Identifier -> Identifier Source|**RECOMMENDED**| Primary identifier for the publicationt. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the publication. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Dimensions|dimensions|**RECOMMENDED**| The different dimensions (granular components) making up a dataset. Providing dimensions give more details about the data types.|The different dimensions (granular components) making up a dataset. Providing dimensions give more details about the data types.|
+|Identifier|identifier|| Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Identifier|Identifier -> Identifier|**RECOMMENDED**| A code uniquely identifying an entity locally to a system or globally.|A code uniquely identifying an entity locally to a system or globally.|
+|Identifier|Identifier -> Identifier Source|**RECOMMENDED**| Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Contact|extraProperties - contact|**RECOMMENDED**| Contact information for this dataset.|Provide contact information (name and email address) of the person responsible for the dataset.| 
+|Logo|logo|**RECOMMENDED**| Link to or provide a logo to display on your resource landing page.|Link to an URL for the logo or local filename containing the logo.
+|Dates|dates|**OPTIONAL**|  Relevant dates for the dataset. If you provide a date, it must come with a description of the date.|Relevant dates for the publication. If you provide a date, it must come with a description of the date (i.e.: first data collection, last data collection, date of first publication, ...) |<!--will later choose from a pulldown list--> 
+|Produced By|producedBy|**OPTIONAL**| Process which generated a given dataset.|Process which generated a given dataset.|
```

Comment:
Can you elaborate on this? What do you mean by process? What is the difference between this field and the creators field. Might be worth clarifying this.

Reply:
`creators`  is for the people who generated the dataset.
`producedBy` is for the procedure used to generate the dataset, so valid values here might be things like "MRI" or "whole genome sequencing", or something more specific like a particular data-processing algorithm.

The ambiguity in the DATS model names has been an issue for a couple of the people providing our data so this one very much needs to be disambiguated.

Reply:
@emmetaobrien Do you have a suggestion?

Reply:
How about something like that? @emmetaobrien @PapillonMcGill 

```suggestion
|Produced By|producedBy|**OPTIONAL**| Process which generated a given dataset. For example: a pipeline, a software... Note: person's name should go in the Creators fields.|Process which generated a given dataset.|
```